src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -120,7 +120,12 @@ class AccordionPanel extends Component {
                   </Box>
                 )}
               </Button>
-              <Box border={{ side: 'bottom', color: 'border' }}>
+              <Box
+                border={{
```

Comment:
I'm wondering if we should put all of this in the theme?
i.e. `<Box border={theme.accordion.border}>`
with the theme having `theme.accordion.border = { side: 'bottom', color: 'border' }`

Reply:
So I was talking to @ShimiSun and the border side is always the bottom because of the way the box is under the button and does not have a height so I was not sure if the line was in the design and should be there and just change the color or like you mentioned made the whole thing a theme

Reply:
We've been steadily moving towards putting these sorts of things into the base theme instead of hard coding them. If you put all of border in the theme, it lets the theme put a full box around, instead of just on the bottom. Or, maybe the theme wants no border at all and instead have a background on the panel header.

Reply:
I see your point Ill get to work! changing it

Reply:
@ericsoderberghp  I wanted your suggestion on how I redid the theme for the border I added none for a option for side so that the user can put none if they do not want the border there since border is a prop of box and it is set there by default? Im open to any other suggestions

Reply:
I don't think `none` is needed. I think callers can set `border: undefined` in their theme to get the same effect.

Reply:
Sounds good I took it out and changed the example to for the story book to show border: undefined to show as a option.